streaming/src/main/scala/org/apache/spark/streaming/api/java/JavaDStreamLike.scala

```diff
@@ -308,7 +308,10 @@ trait JavaDStreamLike[T, This <: JavaDStreamLike[T, This, R], R <: JavaRDDLike[T
   /**
    * Apply a function to each RDD in this DStream. This is an output operator, so
    * 'this' DStream will be registered as an output stream and therefore materialized.
+   *
+   * @deprecated  As of release 2.0.0, replaced by foreachRDD(JVoidFunction)
    */
+  @deprecated("Use foreachRDD(foreachFunc: JVoidFunction[R])", "2.0.0")
```

Comment:
nit: the doc of `since` is `a string identifying the first version in which the definition was deprecated`. It's supposed to be `1.6.0`.